src/App.tsx

```diff
@@ -17,18 +33,24 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                selectAll={selectAll}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {loader && <Loader />}
+              <TodoList
+                todos={todo}
+                setChoiceTodo={(newChoiceTodo) => setChoiceTodo(newChoiceTodo)}
```

Comment:
```suggestion
                setChoiceTodo={setChoiceTodo}
```
this function wrapper is redundant in this case
so just write a reference to function